quiche/src/recovery/congestion/delivery_rate.rs

```diff
@@ -168,8 +172,8 @@ impl Rate {
         self.delivered
     }
 
-    pub fn sample_delivery_rate(&self) -> u64 {
-        self.rate_sample.delivery_rate
```

Comment:
So apparently delivery rate and bandwidth are slightly different concepts...

The name delivery_rate seems more descriptive name that refers to something in the BBR draft.  Bandwidth may be an ok name for the struct, but it may make sense to use delivery rate for many of the function and data member names because it tells us more about where the bandwidth measurement came from.

Reply:
Yep that makes sense.